discovery-api/templates/deployment.yaml

```diff
@@ -19,26 +19,26 @@ spec:
       containers:
       - name: discovery-api
         image: {{ .Values.image.name }}:{{ .Values.image.tag }}
-        imagePullPolicy: Always
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
```

Comment:
Do we default this anywhere?

Reply:
Yup this was in the values file already, just wasn't referenced